vowpalwabbit/cb_explore_adf.cc

```diff
@@ -260,6 +260,7 @@ void predict_or_learn_first(cb_explore_adf& data, multi_learner& base, multi_ex&
 template <bool is_learn>
 void predict_or_learn_greedy(cb_explore_adf& data, multi_learner& base, multi_ex& examples)
 {
+	data.offset = examples[0]->ft_offset;
```

Comment:
Why do we set something which is never used?

Reply:
I believe we need this additional line - we had an email discussion on this on July 7, 2018:

If we don't have this line, in warm_cb.cc, base.learn(data.adf_data.ecs, 0) / base.learn(data.adf_data.ecs, 1)'s updates seem to be on the same set of weights.